js/index.js

```diff
@@ -67,6 +67,9 @@ const getNumOfSameDiceValues = sortedArr => {
 // TO DO
 const getSameArrElements = (arrToSearch, arrSearchItems) => {};
 
+// helper function to map score values to labels
+const mapScoreLabel = (value) => (value[0].toUpperCase() + value.slice(1)).replace(/-/g, ' ');
+
```

Comment:
Can't we use `Game.scoreHistory` for this?

Reply:
Well, we are; the `mapScoreLabel` function just converts the object key into a more legible string.

Reply:
Yes, you are right.